### PR TITLE
ready for review

### PR DESCRIPTION
**Pull request by Habenzy**

### CODE REVIEW

File: public/lessons/agile/acceptance_testing.md [hunk 1]
```diff
@@ -165,7 +165,17 @@ see more at the [Cypress Lesson](/lessons/javascript/cypress)
 * more detailed assertions are written in [Mocha](https://docs.cypress.io/guides/references/bundled-tools.html#Mocha) + [Chai](https://www.chaijs.com/api/bdd/)
   * (very similar to Jasmine and Jest -- but [not identical](https://medium.com/@NicholasBoll/using-jest-matchers-in-cypress-5e8e7281f5dd) )
 
-see more at the [Cypress Lesson](/lessons/javascript/cypress)
+see more at the [Cypress Lesson](/lessons/javascript-track/cypress)
```
Comment (ncastle) [inline public/lessons/agile/acceptance_testing.md:168]: are these links right? following the link gives me a page not found

File: public/lessons/agile/acceptance_testing.md [hunk 2]
```diff
@@ -165,7 +165,17 @@ see more at the [Cypress Lesson](/lessons/javascript/cypress)
...
+
+# Lab: Let's Test Our Projects
```
Comment (ncastle) [inline public/lessons/agile/acceptance_testing.md:170]: This is a cool lab that incorporates one of the projects. Nice

File: public/lessons/client_side_coding/browser_debugging.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+ 
+# Step 2a: Add Breakpoints in Chrome
+
+* In the dev tools, navigate to the `Sources` tab (to the immediate right of the `console` tab)
+
+* Select the file you want to debug
+
+* Click on the line number where you want to add your breakpoint
+
+>Note: If you're unsure where your error is coming from you can add abreak point at the top of your code and step through your whole program line by line
```
Comment (ncastle) [inline public/lessons/client_side_coding/browser_debugging.md:35]: add `a breakpoint`

File: public/lessons/client_side_coding/embedding.md [hunk 4]
```diff
@@ -10,26 +21,123 @@ For instance, you could programmatically pause and play an embedded audio clip,
...
+
+# Embedding Video
+
+HTML5 defines a standard `<video>` tag
+
+  * Use the `<video>` tag when you are hosting your own video media file (on your web server or a CDN)
+
+  * video hosting sites like YouTube, and Vimeo have their own rules and sample code which you should find and copy into your HTML file
+
+> **Tip**: you can use automatically start playing the video when the page loads; to be polite you should also mute the volume, like this: `<video muted=true autoplay=true src='/videos/yelling-man.mp4'>`
```
Comment (ncastle) [inline public/lessons/client_side_coding/embedding.md:101]: you can ~~use~~ automatically

File: public/lessons/client_side_coding/embedding.md [hunk 5]
```diff
@@ -10,26 +21,123 @@ For instance, you could programmatically pause and play an embedded audio clip,
...
+
+* https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video
+* https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Video_and_audio_content
+
+# Embedding Audio
+
+HTML5 also has a prebuilt `<audio>` tagthat you can use to embed your local audio files into your web pages. There are several key attributes for audio elements.
```
Comment (ncastle) [inline public/lessons/client_side_coding/embedding.md:108]: `tag that`

File: public/lessons/www/forms.md [hunk 6]
```diff
@@ -6,7 +6,7 @@
 
 # Forms contain inputs
 
-a `<form>` is an HTML element that contains input elements
+a `<form>` is an HTTP element that contains input elements
```
Comment (ncastle) [inline public/lessons/www/forms.md:9]: HTTP?

Comment (Habenzy) [reply public/lessons/www/forms.md:9]: Whoops!

**Pull request merged**